app/views/trainees/degrees/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: [ @trainee, @degree ], local: true do |f| %>
+      <h2 class="govuk-heading-l"> Degree Details </h2>
+      <%= f.govuk_error_summary %>
+      <%- degree_subjects = DEGREE_SUBJECTS.map do |name| 
```

Comment:
same for this one, should it be in a helper to keep the view dumb?

Reply:
next pr @JonF1